CONTRIBUTING.md

```diff
@@ -77,6 +77,57 @@ git remote add upstream git@github.com:virtual-kubelet/virtual-kubelet.git
 ```
 3. Submit a pull request.
 
+
+## Submission and Review guidelines
+
+We welcome and appreciate everyone to submit and review changes. Here are some guidelines to follow for help ensure
+a successful contribution experience.
+
+Please note these are general guidelines, and while they are a good starting point, they are not specifically rules.
+If you have a question about something, feel free to ask.
+
+#### Don't make breaking API changes.
+
+Since Virtual Kubelet has reached 1.0 it is a major goal of the project to keep a stable API.
```

Comment:
I feel like this is a nicer way of saying don't fuck it up. I think one thing we learned from the events change for example, is that simple changes, even though API compatible can result in massive cascading failures due to change in behaviour of the system.

I think that saying be careful about changing the behaviour of the system, and if you're worried it might impact the system, make it clear how it'll change the behaviour of the system, and consider adding a knob to change the behaviour.

Reply:
Yep, I will add more detail on that.

Do we need to fix the events change? 😄

Reply:
Added more detail here.

Reply:
We ended up tactically fixing it by hacking around it with a custom event recorder. I believe in the PR I asked if we could put it behind a flag, and there was no flag. :(
https://github.com/virtual-kubelet/virtual-kubelet/pull/797#issuecomment-572682283